### ISSUE TITLE
Naming conventions

### ISSUE THREAD

**Issue by jolleekin** [DavBfr/dart_pdf#23]
According to [Dart's style guidelines](https://www.dartlang.org/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words), acronyms and abbreviations longer than two letters should be treated as words
```
// GOOD
PdfDocument
PdfTtfFont
```
```
// BAD
PDFDocument
PDFTTFFont
```

**Comment by DavBfr** [comment]
Thanks. This will be in Version 2.0

**Comment by DavBfr** [comment]
Fixed in version 1.1.0
